mkdocs/docs/tutorials/traffic-engineering/session-affinity.md

```diff
@@ -0,0 +1,179 @@
+    ```
+
+    ??? info "Look inside experiment.yaml"
+        ```yaml linenums="1"
+        apiVersion: iter8.tools/v2alpha2
+        kind: Experiment
+        metadata:
+          name: session-affinity-exp
+        spec:
+          target: flowers
+          strategy:
+            testingPattern: A/B
+            deploymentPattern: FixedSplit
+            actions:
+              # when the experiment completes, promote the winning version using kubectl apply
+              finish:
+              - task: common/exec
+                with:
+                  cmd: /bin/bash
+                  args: [ "-c", "kubectl apply -f {{ .promote }}" ]
+          criteria:
+            requestCount: iter8-kfserving/request-count
+            rewards: # Business rewards
+            - metric: iter8-kfserving/user-engagement
+              preferredDirection: High # maximize user engagement
+            objectives:
+            - metric: iter8-kfserving/mean-latency
+              upperLimit: 2000
+            - metric: iter8-kfserving/95th-percentile-tail-latency
+              upperLimit: 5000
+            - metric: iter8-kfserving/error-rate
+              upperLimit: "0.01"
+          duration:
+            intervalSeconds: 10
+            iterationsPerLoop: 10
+          versionInfo:
+            # information about model versions used in this experiment
+            baseline:
+              name: flowers-v1
+              variables:
+              - name: ns
+                value: ns-baseline
+              - name: promote
+                value: https://raw.githubusercontent.com/iter8-tools/iter8/master/samples/kfserving/quickstart/promote-v1.yaml
+            candidates:
+            - name: flowers-v2
+              variables:
+              - name: ns
+                value: ns-candidate
+              - name: promote
+                value: https://raw.githubusercontent.com/iter8-tools/iter8/master/samples/kfserving/quickstart/promote-v2.yaml
+        ```
+
+The process automated by Iter8 during this experiment is depicted below.
+
+![Iter8 automation](../../images/session-affinity-iter8-process.png)
+
+## 8. Observe experiment
+Please follow [Step 8 of the quick start tutorial](../../../getting-started/quick-start/#8-observe-experiment) to observe the experiment in realtime. Note that the experiment in this tutorial uses a different name from the quick start one. Replace the experiment name `quickstart-exp` with `session-affinity-exp` in your commands.
+
+
+???+ info "Understanding what happened"
+    1. You created two versions of your app/ML model.
+    2. You generated requests for your app/ML model versions. Throughout the experiment, users from Group 1 (i.e., users whose requests had a `userhash` header value prefixed with `111` ) were routed to the baseline version; and users from Group 2 (i.e., users whose requests had a `userhash` header value prefixed with `101` ) were routed to the candidate version.
```

Comment:
I wonder if this information could be added earlier in the tutorial. I was wondering what was being done to identify users. Perhaps this could be part of the version setup description?